Makefile.js

```diff
@@ -282,6 +282,13 @@ function generateRelease() {
     generateBlogPost(releaseInfo);
     commitSiteToGit(`v${releaseInfo.version}`);
 
+    echo("Updating version in docs package");
+    const docsPackagePath = path.join(__dirname, "docs", "package.json");
+    const docsPackage = require(docsPackagePath);
+
+    docsPackage.version = releaseInfo.version;
+    fs.writeFileSync(docsPackagePath, JSON.stringify(docsPackage, null, 4));
```

Comment:
```suggestion
    fs.writeFileSync(docsPackagePath, `${JSON.stringify(docsPackage, null, 4)}\n`);
```

To keep a newline at the end.

Reply:
Done